resources/metastatus.py

```diff
@@ -1,16 +1,14 @@
-import falcon
-import json
-
-from .base import BaseResource
-
-
-class Metastatus(BaseResource):
-
-
-
-    def on_get(self, req, res):
-        
-        res.status = falcon.HTTP_200
-        res.body = json.dumps({
-                'msg': 'pong.'
+import falcon
+import json
+
+from .base import BaseResource
+
+
+class Metastatus(BaseResource):
```

Comment:
Call this class `HealthCheck`! Even a healthcheck service needs its own healthcheck.

Reply:
The other one should be a probe of health check. I'd prefer calling this one `SelfHealth` or something